src/main/java/com/amazon/dlic/util/RolesUtil.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.dlic.util;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class RolesUtil {
+    private static final Logger log = LogManager.getLogger(RolesUtil.class);
+
+    /*
+     * Split the given rolesObject to one or several roles
+     * We accept String or JSONarray only
+     *
+     * rolesObject should be String if given by roles_key
+     * rolesObject should be String or ArrayList if given by roles_path
+     *
+     * Here are some examples:
+     *
+     *   JWT paylaoad contains:
+     *     "access": {
+     *       "roles": [
+     *         "readall, testrole",
+     *         "admin",
+     *         123,
```

Comment:
what is the meaning of `123` for the role?

Reply:
This is to specify that non-string type is ignored

Reply:
If roles contains `123`, why is it ignored? Under what condition roles will include non-string types?

Reply:
I had a discussion with @palashhedau, and we came up with the logic that if the input is a json array, collect all element that is a string (whether it contains comma or not, we treat it as a whole single string) and discard otherwise. If the input is a string, we separate the string with comma.

Reply:
@palashhedau When roles will contain anything other than a string or a comma separated string? Why is it OK to ignore such elements in the roles instead of erroring out? Why some elements are strings and some are comma separated strings?

Reply:
Its up to the user how they configure their JWT payload. 
Either it should be a string or array of string. If it is a string, it will behave same as what it behaved before (split by comma). 

If it is an array of element, we just want to make sure elements are string. (In this case, we dont split by comma because it doesnt make sense)

There could be a payload when they have nested roles, but we will support only string elements in the roles_path

Reply:
Why some elements are strings and some are comma separated strings?
> Its just a test added by Andy. If roles_path is an array, it should just take strings elements. if roles_path is string, then split by comma

Reply:
I don't think it is a good idea to silently skip invalid entries. If it is not an error, at least it should be logged as a warning.

Reply:
Agree, I will add warning something like "We only accept String for elements in JSON array, 123 is not a String"